src/components/ImageSlider.vue

```diff
@@ -33,52 +33,55 @@ export default {
         this.currentSlide = this.images.length - 1;
       }
     },
+    selectSlide(index: number): void {
+      this.currentSlide = index;
+    },
```

Comment:
Since the methods `nextSlide()`, `previousSlide()`, and `selectSlide(index: number)` do not modify any data properties, you can declare them as `const` functions instead of methods. This makes it clear that they are read-only and can improve performance.